packages/nouvelle/docs/index.md

```diff
@@ -7,7 +7,9 @@ position: 1
 
 # Nouvelle Theme Overview
 
-> Kendo UI Nouvelle Theme is currently in development stage and it is, currently, covering the base components. Please vote for it if you want a full coverage: https://feedback.telerik.com/themes/1432136-add-theme-based-on-css-variables.
+> Kendo UI Nouvelle Theme is currently experimental and covers the base components such as buttons, inputs, etc. We have an official feature request that you can vote for if you want a full coverage: https://feedback.telerik.com/themes/1432136-add-theme-based-on-css-variables. Once scoped and planned, we will continue with the implementation. The updates will be posted in [this public issue](https://github.com/telerik/kendo-themes/issues/798).
+
+In is root, the Kendo UI Nouvelle Theme is a CSS variables based for the Kendo UI components. [CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) are properties you set in your stylesheets that reuse throughout the application.
```

Comment:
In its essence, the Kendo UI Nouvelle Theme is a theme based on CSS variables that can be used with the Kendo UI component libraries. [CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) are properties you set in your stylesheets that cab be reused throughout the application.

Reply:
Could you add a note/phrase about what the other Kendo UI themes are based on so people can better understand what's different abut this theme?